pkg/cmd/show.go

```diff
@@ -273,7 +264,9 @@ func showPrometheus(targetReader TargetReader) {
 func showAltermanager(targetReader TargetReader) {
 	username, password = getMonitoringCredentials()
 	showPod("alertmanager", "seed", targetReader)
-	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "alertmanager", "-n", getFromTargetInfo("shootTechnicalID"))
+	// fmt.Print(targetInfo)
+	// os.Exit(0)
```

Comment:
remove
```suggestion
```

Reply:
forget remove it same as part as below .done
if isTargeted("seed", "shoot") {